alertwildfire_live_recording.py

```diff
@@ -68,23 +68,30 @@ def capture_and_record(googleServices, outputDir, camera_name):
     Returns:
         imgPath: local path to downloaded object
     """
-    imgPath = alertwildfire_API.request_current_image(outputDir, camera_name)
-    cloud_file_path = camera_name + '/' + pathlib.PurePath(imgPath).name
-    goog_helper.uploadBucketObject(googleServices["storage"],"fuego-firecam-a",cloud_file_path, imgPath)
+    success = False
+    while not success:
```

Comment:
It's usually better to limit the number of retries.  In this case maybe 5?
Also, in case of mismatch, you could make it slightly more efficient by setting pull1 = pull2 and move the pull1 line before the while loop.